### ISSUE TITLE
No twemoji parsing in Richtext::preview() (Activities/Mail/Notification)

### ISSUE THREAD

**Issue by binarynos** [humhub/humhub#3520]
### What steps will reproduce the problem?
Here are some screenshots to explain the issue:
<br>
Example of an emoji used in a comment. The emoji character has been replaced with an `<img>`
![hk4nt](https://user-images.githubusercontent.com/21215818/54470839-19533380-47d5-11e9-8fb8-526f0ccfb59b.jpg)

Example of an emoji in the latest activities sidebar.  Just the emoji character is shown. In this case different OS will render the emojis differently. 
![uWdoz](https://user-images.githubusercontent.com/21215818/54470860-8bc41380-47d5-11e9-88ca-26bc222141d3.jpg)

### What is the expected result?
The emoji character should get replaced with `<img>`

### What do you get instead?
See screenshot two.
### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.3.8
| PHP version      |  7.2
| Operating system | Any OS

**Comment by kesselb** [comment]
> The emoji character should get replaced with <img>

Yes the emoji looks different on each os but there is nothing wrong with it. This is true for emails, sms, etc. Why should humhub do something different?

**Comment by binarynos** [comment]
> Why should humhub do something different?
Consistency. The emojis should look the same on all devices regardless of the OS.  This is why HH uses twemoji, right?

**Comment by kesselb** [comment]
> Consistency. The emojis should look the same on all devices regardless of the OS. This is why HH uses twemoji, right?

Fair enough. @buddh4 :ping_pong:

**Comment by acs-ferreira** [comment]
IMHO, we should **remove Twemoji** completely.

And use only this in the font-family CSS: `"Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"`

https://github.com/twbs/bootstrap/blob/091aa1e9fd411440c3ff3c95042a5aa1502e6978/dist/css/bootstrap.css#L57

This would ensure consistency and also be supported on local installations.

**Comment by buddh4** [comment]
The emojis should be fetched from the local server instead of the cdn anyways, this is rather a problem with the richtext which uses the wrong url. The twemoji solution would allow replacing and perhaps extending the emoji set in the future.
